packages/twenty-front/src/pages/settings/data-model/SettingsObjectOverview/EdgeUtil.ts

```diff
@@ -0,0 +1,60 @@
+export const calculateSourcePosition = (
+  sourceNodeWidth: number,
+  sourceNodeX: number,
+  targetNodeWidth: number,
+  targetNodeX: number,
+) => {
+  if (sourceNodeX > targetNodeX + targetNodeWidth) {
+    return 'left';
+  } else if (
+    sourceNodeX > targetNodeX &&
+    sourceNodeX < targetNodeX + targetNodeWidth
+  ) {
+    return 'right';
+  } else if (sourceNodeX + sourceNodeWidth > targetNodeX) {
+    return 'left';
+  }
+  return 'right';
+};
+export const calculateTargetPosition = (
```

Comment:
so this should go to modules/settings/data-model/graph-overview/utils/calculateEgeTargetPosition.ts for example

We also do one util per file, and it would be great to add unit tests

Reply:
you can have un exported sub-function in the same file though (~private ones)